packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -1093,6 +1093,24 @@ class WKWebView extends UIView {
     );
   }
 
+  /// Enables debugging of web contents (HTML / CSS / JavaScript) in the
+  /// underlying WebView.
+  ///
+  /// This flag can be enabled in order to facilitate debugging of web layouts
+  /// and JavaScript code running inside WebViews. Please refer to [WKWebView]
```

Comment:
Could you add the link to the WKWebView documentation you want to point to? Like:
[WKWebView](link)

Reply:
@cyanglaz 

I added a link to the document.

4629c89b12cc4a9bc6edd8ad24cfdf84217af378